.github/workflows/test.yml

```diff
@@ -19,3 +19,12 @@ jobs:
 
     - name: Run Unit Tests
       run: npm run test
+
+    - name: Run E2E Tests
```

Comment:
Should we run the E2E tests in their own github action separate from unit tests?

Will help in reporting in identifying what is the error more quickly

Reply:
makes sense, I created a workflow file for e2e tests.